src/context-menu/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import { createICacheMiddleware } from '@dojo/framework/core/middleware/icache';
+import theme from '../middleware/theme';
+import Menu, { MenuOption } from '../menu/index';
+import * as menuCss from '../theme/default/menu.m.css';
+import * as css from '../theme/default/context-menu.m.css';
+import * as fixedCss from './context-menu.m.css';
+
+export interface ContextMenuProperties {
+	options: MenuOption[];
+	onValue(value: string): void;
+	menuWidth?: number;
```

Comment:
Do we really need `menuWidth` here? The `popup` widget uses `dimensions` to figure out the size of the menu where appropriate.

Reply:
I'm not sure using `dimensions` would make sense for a context menu. In the example the context menu can be applied to any region of text. In general you could be applying the context menu to any kind of content, and unlike the popup which sizes to the specific content it's attached to I would think that the context menu's width should be consistent regardless of what component is clicked on?